app/controllers/api/v1/scholarship_applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class ScholarshipApplicationsController < ApplicationController
+      respond_to :json
+
+      def create
+        @scholarship_application = ScholarshipApplication.new(reason: params[:reason], terms_accpeted: params[:terms_accepted], scholarship_id: params[:scholarship_id], user_id: current_user.id)
```

Comment:
Creative, you could do that.  It feels like a little misdirection IMO.  

What about building the `ScholarshipApplication` object directly off the `current_user`?  Like this:

```ruby
current_user.scholarship_applications.create! scholarship_application_params
```